src/defaultsettings.cpp

```diff
@@ -534,6 +534,7 @@ void set_default_settings()
 	settings->setDefault("server_address", "");
 	settings->setDefault("server_name", "");
 	settings->setDefault("server_description", "");
+	settings->setDefault("server_announce_send_players", "true");
```

Comment:
i have one question, should we keep this for "compat" (as there is no compat issue), or we should by default keep "privacy" and set it false ?
i'm more in the favor of the second one, and be opt-in

Reply:
Before this change names were sent by default, so there is "compat" situation. However I don't see it as operation critical to keep this ``true``. It makes sense to me to default to ``false``.

Reply:
I would prefer to keep compatibility. Many servers were probably happy with / unaware of the current situation). If they were aware, they would've already done it (like LinuxForks).